frontend/src/metabase/dashboard/actions/utils.unit.spec.ts

```diff
@@ -201,4 +206,57 @@ describe("dashboard > actions > utils", () => {
       expect(endTime - startTime).toBeLessThan(100); // 100 ms (locally this was 6 ms)
     });
   });
+
+  describe("getDashCardMoveToTabUndoMessage", () => {
```

Comment:
Testing the thunk involved too much mocking so I tested the util function.

The whole functionality is also tested in the e2e for two cases, a named card and a text card so we should be covered